components/LoginSection/LoginSection.jsx

```
@@ -43,38 +45,54 @@ const LoginSection = () => {
+      redirect ||
+      process.env.NEXT_PUBLIC_MAIN_SITE ||
+      "https://www.hemocione.com.br/";
+
+    const url = new URL(locationRedirect);
+    if (url.hostname.endsWith("hemocione.com.br")) {
+      url.searchParams.append("token", response.data.token);
+    }
+    const newLocationRedirect = url.toString();
+
+    window.open(newLocationRedirect, "_self");
+  }
```

ESLint

[error] Insert `;`

(prettier/prettier)